continuous_integration/circle/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -ex
+
+# -- Install Miniconda
+MINICONDA=Miniconda3-latest-Linux-x86_64.sh
+wget https://repo.continuum.io/miniconda/$MINICONDA -O miniconda.sh
+bash miniconda.sh -b -p $HOME/miniconda
+source $HOME/miniconda/etc/profile.d/conda.sh
+
+# -- Install Yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update && sudo apt-get install yarn
+export PATH="$PATH:/opt/yarn-1.16.0/bin"
+
+# -- Make new conda environment with required Python version
+conda create -y -n test python=$PYTHON_VERSION
+conda activate test
+
+# Install dependencies
+conda install -q -y -c spyder-ide --file requirements/conda.txt
+
+# Install test dependencies
+conda install -q -y -c spyder-ide --file requirements/tests.txt
+
+# Install qt
+pip install pyqt5==5.11.*
```

Comment:
This is wrong too. The thing is conda and pip packages are binary incompatible so they can't be mixed like this.

What's the problem with the conda pyqt package?

Reply:
Yes this is a mistake! I'm going to install this version using conda. Previously, I used the version 5.9.6 and the segmentation fault error persisted.